contracts/proofs/ExitValidator.sol

```diff
@@ -20,9 +19,8 @@ contract ExitValidator is RootChainValidator {
 
     bytes txBytes,
     bytes txProof
-  ) public {
-    // validate exit id and get owner
-    address owner = validateExitId(exitId);
+  ) external {
```

Comment:
`External functions are sometimes more efficient when they receive large arrays of data.` For public functions these arrays need to be copied to be memory whereas external functions can read directly from calldata. Since this function has several bytes arrays, making it external might make sense.

Reply:
Interestingly, making it external fails with `/contracts/contracts/proofs/ExitValidator.sol:53:37: CompilerError: Stack too deep, try removing local variables.` - when actually it should be using lesser stack memory! Will dig in.

Reply:
Apparently, bytes types in calldata need two stack slots, while they only need a single stack slot in memory. See (this)[https://github.com/ethereum/solidity/issues/5751]